### PR TITLE
add ability to define subscription and topic defaults in console

### PR DESCRIPTION
**Pull request by adamdubiel**

### CODE REVIEW

File: hermes-console/static/js/console/topic/TopicFactory.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+var topics = angular.module('hermes.topic.factory', []);
+
+topics.factory('TopicFactory', ['TOPIC_CONFIG',
+    function(topicConfig) {
+        return {
+            create: function() {
+                var defaults = {retentionTime: {duration: 1}, contentType: 'JSON', ack: 'LEADER'};
+                _.merge(defaults, topicConfig.defaults);
```
Comment (dankraw) [inline hermes-console/static/js/console/topic/TopicFactory.js]: Is it working when `topicConfig.defaults` is `undefined`?

**Comment by dankraw** [comment]
👍

File: hermes-console/static/js/console/topic/TopicFactory.js (hunk 1 above)
Comment (adamdubiel) [inline hermes-console/static/js/console/topic/TopicFactory.js]: yes

**Comment by wojtkiewicz** [comment]
👍

**Comment by adamdubiel** [comment]
branch change

**Pull request closed without merging**